tags/3.0/_index.md

```diff
@@ -48,7 +51,7 @@ The following TCK updates will be required:
 * [Jakarta Standard Tag Library 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.jstl/releases/3.0.0)
 
 # Compatible Implementations
-* [EE4J implementation of Jakarta Standard Tag Library 3.0.0](https://github.com/eclipse-ee4j/jstl-api/releases/download/3.0.0-IMPL-RELEASE/jakarta.servlet.jsp.jstl-3.0.0.jar) using vehicle [Eclipse Glassfish x.0.0-x](https://github.com/eclipse-ee4j/glassfish/releases/download/)
+* [EE4J implementation of Jakarta Standard Tag Library 3.0.0](https://github.com/eclipse-ee4j/jstl-api/releases/tag/3.0.0-IMPL-RELEASE)
```

Comment:
FYI: Updated this to say our latest impl (as that's what was used against the TCK)